src/core/software/pupil-service.md

```diff
@@ -4,76 +4,37 @@ description: Pupil Service is a GUI-less real-time eye tracking software used pr
 </div>
 
-Pupil Service is designed to run in the background and to be controlled via network commands only. The tools introduced in the [hmd-eyes project](https://github.com/pupil-labs/hmd-eyes) are made to work with Pupil Service and Pupil Capture alike.
+:::tip
+<v-icon large color="info">info_outline</v-icon>
+The tools introduced in the [hmd-eyes project](https://github.com/pupil-labs/hmd-eyes) are made to work with Pupil Service and Pupil Capture alike.
+:::
 
-## Talking to Pupil Service
-Code examples below demonstrate how to control Pupil Service over the network.
-
-```
-Starting and stopping Pupil Service:
-```
-
-```python
-import zmq, msgpack, time
-ctx = zmq.Context()
-
-#create a zmq REQ socket to talk to Pupil Service/Capture
-req = ctx.socket(zmq.REQ)
-req.connect('tcp://localhost:50020')
-
-#convenience functions
-def send_recv_notification(n):
-    # REQ REP requirese lock step communication with multipart msg (topic,msgpack_encoded dict)
-    req.send_multipart(('notify.%s'%n['subject'], msgpack.dumps(n)))
-    return req.recv()
-
-def get_pupil_timestamp():
-    req.send('t') #see Pupil Remote Plugin for details
-    return float(req.recv())
-
-# set start eye windows
-n = {'subject':'eye_process.should_start.0','eye_id':0, 'args':{}}
-print(send_recv_notification(n))
-n = {'subject':'eye_process.should_start.1','eye_id':1, 'args':{}}
-print(send_recv_notification(n))
-time.sleep(2)
-
-
-# set calibration method to hmd calibration
-n = {'subject':'start_plugin','name':'HMD_Calibration', 'args':{}}
-print(send_recv_notification(n))
-
-
-time.sleep(2)
-# set calibration method to hmd calibration
-n = {'subject':'service_process.should_stop'}
-print(send_recv_notification(n))
-```
-
-## Notifications
-The code demonstrates how you can listen to all notification from Pupil Service. This requires a little helper script called [zmq_tools.py](https://github.com/pupil-labs/pupil/blob/master/pupil_src/shared_modules/zmq_tools.py).
+## Service Features
+With the Pupil Service GUI you can define a [Pupil Remote Address](/developer/core/network-api/#pupil-remote)
+and toggle to open and close the left and right eye windows.
 
+<div class="pb-4">
+  <img src="../../media/core/imgs/ps-gui.jpg" style="display:flex;margin:0 auto;width:400px">
+</div>
 
-```python
-from zmq_tools import *
+Note that the eye windows opened by Pupil Service are equivalent to those of Pupil Capture, giving you access to all
+associated [pupil detection settings](/core/software/pupil-capture/#fine-tuning-pupil-detection).
 
-ctx = zmq.Context()
-requester = ctx.socket(zmq.REQ)
-requester.connect('tcp://localhost:50020') #change ip if using remote machine
+A powerful feature of Pupil Service is low latency gaze mapping – Pupil Service maps gaze as soon as a new pupil datum 
+is available. Pupil Capture's event loop, on the other hand, is driven by the scene camera, where pupil data is mapped in 
+bulk once the scene camera returns a new frame. 
```

Comment:
I had an idea on how to formulate this more accurately without having to name the event loop. I do not like the repeated "queue*". Any ideas to improve that?
```suggestion
is available. Pupil Capture, on the other hand, queues incoming pupil data until the scene camera returns a new frame and maps the queued pupil data in bulk afterward. 
```